lemur/common/utils.py

```diff
@@ -504,7 +504,7 @@ def drop_last_cert_from_chain(full_chain: str) -> str:
     pem_certificate = OpenSSL.crypto.dump_certificate(
         OpenSSL.crypto.FILETYPE_PEM,
         OpenSSL.crypto.load_certificate(
-            OpenSSL.crypto.FILETYPE_PEM, ''.join(cert.decode() for cert in full_chain_certs[:-1])
+            OpenSSL.crypto.FILETYPE_PEM, ''.join(cert.decode() for cert in full_chain_certs[:-1]).encode()
```

Comment:
That is the only part I would appreciate additional clarification or context on.

Reply:
`load_certificate` takes a byte string. Before we were passing a unicode string, but that hasn't caused any issues since PEM only uses ASCII characters. `encode` converts a str to a bytes in python so the type matches